lib/msal-core/src/MsalTypes.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Key-Value type to support queryParams and extraQueryParams
+ */
+export type Dict = {[key: string]: string};
```

Comment:
i am fine with this but dict is like a hashmap, which can be key valued by any Generic, Dict<T, T> where as this is string specific.

I wonder if what we want is the typescript object type

Reply:
Any name suggestions?

Reply:
Any ideas for naming?